### PR TITLE
Try to address https://github.com/solid/solid-ui/issues/441

### PR DESCRIPTION
**Pull request by timbl**
This removed the PUT and leaves only a PATCH which should work.

### CODE REVIEW

**Comment by timbl** [comment]
See https://github.com/solid/solid-ui/issues/441

File: .web_base [hunk 1]
```diff
@@ -0,0 +1 @@
+https://github.com/solid/solid-ui/tree/main
```
Comment (RubenVerborgh) [inline .web_base:1]: Do we need this document?

**Review by RubenVerborgh — APPROVED** [review]
Tested with the CSS Mashlib recipe and confirmed to be working.

File: src/chat/infinite.js [hunk 2]
```diff
@@ -209,6 +209,7 @@ export async function infiniteMessageArea (dom, kb, chatChannel, options) {
           chatDocument
         )
         await createIfNotExists(chatDocument)
+        // Otherwise, create the documentif necessary with a PATCH:
```
Comment (TallTed) [inline src/chat/infinite.js:212]:
```suggestion
        // Otherwise, create the document, if necessary, with a PATCH:
```

**Pull request merged**